addons/website/models/website.py

```diff
@@ -1235,7 +1235,7 @@ def _enumerate_pages(self, query_string=None, force=False):
 
                     for rec in converter.generate(uid=self.env.uid, dom=query, args=val):
                         newval.append(val.copy())
-                        newval[-1].update({name: rec})
+                        newval[-1].update({name: rec.with_context(lang=None)})
```

Comment:
```suggestion
                        newval[-1].update({name: rec.with_context(lang=website.default_lang_id.code)})
```
Otherwise the sitemap is always in English

Reply:
Indeed, it's fixed :heavy_check_mark: